config_to_json.php

```diff
@@ -7,8 +7,14 @@
  *
  */
 
-$init_modules = array();
+$init_modules = array('nodb');
 require __DIR__ . '/includes/init.php';
+try {
```

Comment:
Removed this section entirely mergedb was unable to be imported. Does it do anything?